src/fairly/__init__.py

```diff
@@ -128,10 +147,29 @@ def client(id: str, **kwargs) -> Client:
         raise ValueError("Invalid client id")
     return clients[id](**kwargs)
 
-
 def get_local_dataset(path: str) -> LocalDataset:
     return LocalDataset(path)
+
+
+def set_fairly_default_config():
+    """
+    Check if the ~/.fairly/config.json exists, if not create it
+    """
+    config_dir = os.path.expanduser("~/.fairly/")
+    config_file = os.path.join(config_dir, "config.json")
     
+    if not os.path.exists(config_dir):
+        os.mkdir(config_dir)
+
+    if not os.path.exists(config_file):
+        write_default_config(get_repositories())
+
+    print("Config file created at ~/.fairly/config.json")
+    print("Add your authentication tokens to the config file to interact with the different repository platforms")
+
+# We run this function to create the default config file
+set_fairly_default_config()
+
```

Comment:
This shouldn't be created. Also we should refrain to use print() calls, as this may affect output of software that use the package (e.g. CLI) in a way that is not wanted by the developer.